packages/inter-protocol/src/provisionPoolKit.js

```diff
@@ -311,37 +319,6 @@ export const prepareProvisionPoolKit = (
           );
           facets.helper.publishMetrics();
         },
-        onProvisioned() {
-          const { state, facets } = this;
-          state.walletsProvisioned += 1n;
-          facets.helper.publishMetrics();
-        },
-        /** @param {ERef<Bank>} destBank */
-        async sendInitialPayment(destBank) {
```

Comment:
good catch!  I thought I saw that one of these was being used elsewhere, and so needed to be kept in `helper`, but all the uses are for the handler.